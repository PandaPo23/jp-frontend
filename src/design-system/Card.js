import system from '../utils/System';
import Flex from './Box';

const Card = system('Card', {
  extend: Flex,
  borderRadius: 'sm',
  boxShadow: 4,
  className: 'surface',
  bg: 'surface',
});

export default Card;
