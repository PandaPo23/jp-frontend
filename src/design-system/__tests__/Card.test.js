import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Card from '../Card';

describe('Card component', () => {
  it('should render as card', () => {
    const wrapper = mount(<Card />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
