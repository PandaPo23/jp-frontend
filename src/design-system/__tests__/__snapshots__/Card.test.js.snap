// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should render as card 1`] = `
.emotion-0 {
  background-color: surface;
  border-radius: sm;
  box-shadow: 4px;
}

<Card
  bg="surface"
  borderRadius="sm"
  boxShadow={4}
  className="surface"
>
  <div
    className="surface emotion-0"
  />
</Card>
`;
